Makefile

```diff
@@ -138,16 +176,16 @@ build-cross-platform: eks-a-cross-platform
 
 .PHONY: eks-a-tool
 eks-a-tool: ## Build eks-a-tool
-	go build -o bin/eks-a-tool github.com/aws/eks-anywhere/cmd/eks-a-tool
+	$(GO) build -o bin/eks-a-tool github.com/aws/eks-anywhere/cmd/eks-a-tool
 
 .PHONY: eks-a-cluster-controller
 eks-a-cluster-controller: ## Build eks-a-cluster-controller
-	go build -ldflags "-s -w -buildid='' -extldflags -static" -o bin/manager ./controllers
+	$(GO) build -ldflags "-s -w -buildid='' -extldflags -static" -o bin/manager ./controllers
 
 # This target will copy LICENSE file from root to the release submodule
 # when fetching licenses for cluster-controller
 .PHONY: copy-license-cluster-controller
-copy-license-cluster-controller: SHELL := /bin/bash
```

Comment:
This is no longer needed here since the default shell is not set to /bin/bash at the top of the make file